call_server/templates/layouts/base_admin.html

```diff
@@ -54,9 +54,6 @@
 <script>
     window.CallPower = {
       Config: {
-        SUNLIGHT_API_KEY: "{{SUNLIGHT_API_KEY}}",
```

Comment:
For the front end to work we need to replace this config with the values for openstates.

```js
{
  OPENSTATES_URL: "https://v3.openstates.org/people?include=offices", // need offices in the response to get phone number
  OPENSTATES_API_KEY: "..." // get from app config
}
```